src/actions/guides/database/database_setup.cr

```diff
@@ -0,0 +1,113 @@
+class Guides::Database::DatabaseSetup < GuideAction
+  guide_route "/database/database-setup"
+
+  def self.title
+    "Database Setup"
+  end
+
+  def markdown
+    <<-MD
+    ## Configure
+
+    Ensure that your system is running a [PostgreSQL](https://www.postgresql.org/docs/current/tutorial-install.html) version of at least 9.4 or later.
+
+    > If you're using the postgres app on macOS, be sure to install [CLI tools](https://postgresapp.com/documentation/cli-tools.html).
+
+    ### Standard Options
+
+    To configure Lucky to connect to your database, you can define a `DATABASE_URL` environment variable. (e.g. `postgres://root@localhost:5432/my_database`). To see more options, look in your app's `config/database.cr` file.
```

Comment:
I think DATABASE_URL should come after the config/database.cr explanation. Since the config file is much easier to use. The DATABASE_URL could be explained right after the config options in this section